lisp/l/common.l

```diff
@@ -925,7 +925,17 @@ if pos is bigger than the length of list, item is nconc'ed at the tail"
 (defun minusp (n) (< n 0))
 (defun oddp (n) (logbitp 0 n))
 (defun evenp (n) (not (logbitp 0 n)))
-(defun /= (n1 n2) (not (= n1 n2)))
+(defun /= (n &rest more-numbers)
+  "Return T if no two of its arguments are numerically equal, NIL otherwise."
+  (if more-numbers
```

Comment:
if this is common list specification, we can think about that.

--
◉ Kei Okada


On Thu, Jul 23, 2015 at 6:04 PM, Furushchev <notifications@github.com>
wrote:

> In lisp/l/common.l
> <https://github.com/euslisp/EusLisp/pull/124#discussion_r35302354>:
>
> > @@ -925,7 +925,17 @@ if pos is bigger than the length of list, item is nconc'ed at the tail"
> >  (defun minusp (n) (< n 0))
> >  (defun oddp (n) (logbitp 0 n))
> >  (defun evenp (n) (not (logbitp 0 n)))
> > -(defun /= (n1 n2) (not (= n1 n2)))
> > +(defun /= (n &rest more-numbers)
> > +  "Return T if no two of its arguments are numerically equal, NIL otherwise."
> > +  (if more-numbers
>
> @k-okada <https://github.com/k-okada> ok I fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/euslisp/EusLisp/pull/124/files#r35302354>.
>